Frontend/src/App.tsx

```diff
@@ -8,27 +8,48 @@ function App() {
   const [searchResults, setSearchResults] = useState<string[] | null>(null);
   const [selectedPackage, setSelectedPackage] = useState<string | null>(null);
 
-    // Package Directory
+  // Package Directory
   const [isPackageDirectoryOpen, setIsPackageDirectoryOpen] = useState<boolean>(false);
   const [packageName, setPackageName] = useState<string>('');
   const [packageVersion, setPackageVersion] = useState<string>('');
   const [packageDirectory, setPackageDirectory] = useState<string | null>(null);
-    // Open package directory
+  const [packageIdInput, setPackageIdInput] = useState<string>('');
+
+  // Update Fields
+  const [updateFields, setUpdateFields] = useState<{ content: string | null, url: string | null}>({ content: '', url: '' });
+
+  // Package Rating
+  const [packageRating, setPackageRating] = useState<number | null>(null);
+
+  // Open package directory
   const openPackageDirectory = () => {
     setIsPackageDirectoryOpen(true);
   };
-    // Close package directory
+
+  // Close package directory
   const closePackageDirectory = () => {
     setIsPackageDirectoryOpen(false);
     setPackageName('');
     setPackageVersion('');
     setPackageDirectory(null);
   };
-    // Package Info
+
+  // Package Info
   const submitPackageInfo = () => {
-    setPackageDirectory(`/${packageName}/${packageVersion}`);
+    // Validate package version consists of only numbers
+    if (!/^(\d+(\.\d+)*)?$/.test(packageVersion)) {
+      alert('Error: Version must contain only numbers.');
+      return;
+    }
+    // Validate package ID consists of only numbers
```

Comment:
no? package ID can have letters to. And you don't need to validate the package ID. Just look at the API response and notify the user accordingly